nck/helpers/facebook_helper.py

```diff
@@ -15,160 +15,125 @@
-    "video_plays": ("video_play_actions", "video_view"),
-    "video_plays_100p": ("video_p100_watched_actions", "video_view"),
-    "video_plays_95p": ("video_p95_watched_actions", "video_view"),
-    "video_plays_75p": ("video_p75_watched_actions", "video_view"),
-    "video_plays_50p": ("video_p50_watched_actions", "video_view"),
-    "video_plays_25p": ("video_p25_watched_actions", "video_view"),
-    "age": "age",
-    "gender": "gender",
-    "account_currency": "account_currency",
-    "frequency": "frequency",
-    "buying_type": "buying_type",
-    "video_p100_watched_actions": "video_p100_watched_actions",
-    "video_p75_watched_actions": "video_p75_watched_actions",
-    "video_p25_watched_actions": "video_p25_watched_actions",
-    "video_p50_watched_actions": "video_p50_watched_actions",
-    "video_thruplay_watched_actions": "video_thruplay_watched_actions",
-    "conversions": "conversions",
-    "status": "status",
-    "lifetime_budget": "lifetime_budget",
-    "budget_remaining": "budget_remaining",
-    "name": "name",
-    "id": "id",
-    "start_time": "start_time",
-    "stop_time": "end_time",
-    "daily_budget": "daily_budget",
-    "device_platform": "device_platform",
-    "platform_position": "platform_position",
-    "publisher_platform": "publisher_platform",
-    "impression_device": "impression_device",
-    "link_url_asset": {"value": "website_url"},
-}
-
-
-def get_field_value(row, field):
-    if is_url_asset(field):
-        return extract_special_field(row, field)
-    return (
-        row.get(DESIRED_FIELDS[field], None)
-        if isinstance(DESIRED_FIELDS[field], str)
-        else get_nested_field_value(row, field)
-    )
-
-
-def extract_special_field(row, field):
-    dic = DESIRED_FIELDS[field]
-    return row.get(field, {}).get(dic.get("value"), None)
-
-
-def is_url_asset(field):
-    return field == "link_url_asset"
-
-
-def get_nested_field_value(row, field):
-    if DESIRED_FIELDS[field][0] not in row:
-        return None
-    nested_field = next((x for x in row[DESIRED_FIELDS[field][0]] if x["action_type"] == DESIRED_FIELDS[field][1]), {})
-    return nested_field["value"] if nested_field else None
+BREAKDOWNS = [v for k, v in AdsInsights.Breakdowns.__dict__.items() if not k.startswith("__")]
+
+ACTION_BREAKDOWNS = [v for k, v in AdsInsights.ActionBreakdowns.__dict__.items() if not k.startswith("__")]
+
+def get_action_breakdown_filters(field_path):
+    """
+    Extracts action breakdown filters from a field path,
+    and returns them as a dictionnary.
+    
+    For instance:
+        'actions[action_type:video_view][action_type:post_engagement][action_device:iphone]'
+    returns:
+        {'action_type':['video_view','post_engagement'],
+        'action_device':['iphone']}
+    """
+    filters = {}
+    for path_item in field_path:
+        if ":" in path_item:
+            action_breakdown = path_item.split(":")[0]
+            action_breakdown_value = path_item.split(":")[1]
```

Comment:
Ici c'est suffisant de faire:
```suggestion
            action_breakdown, action_breakdown_value = path_item.split(":")
```
Pour éviter 2 opérations split, mais c'est un détail

Reply:
En effet, c'est corrigé, merci !